src/DAL/repositories/jobRepository.ts

```diff
@@ -73,12 +73,17 @@ export class JobRepository extends GeneralRepository<JobEntity> {
     } catch (err) {
       const pgExclusionViolationErrorCode = '23P01';
       const error = err as Error & { code: string };
-      if (error.code === pgExclusionViolationErrorCode && error.message.includes('UQ_uniqueness_on_active_tasks')) {
-        this.appLogger.log('info', `failed to create job  because another active job exists for the same resource`);
-        throw new EntityAlreadyExists(`another active job exists for the same resource`);
-      } else {
-        throw err;
+      if (error.code === pgExclusionViolationErrorCode) {
+        if(error.message.includes('UQ_uniqueness_on_active_tasks')){
+          this.appLogger.log('info', `failed to create job  because another active job exists for the same resource`);
+          throw new EntityAlreadyExists(`another active job exists for the same resource`);
+        }
+        if(error.message.includes('UQ_uniqness_on_job_and_type')){
+          this.appLogger.log('info', `failed to create job  because it contains duplicate tasks.`);
```

Comment:
Warn log, add job id, task relevant paremters to log

Reply:
I don't have the job id

Reply:
you are right in case of a new job, but if it's tasks of an existing jobs?
either way add the resourceId, version and type and any other relevant data

Reply:
this is the "creatJob" API

Reply:
ok, just mind the 2'cnd section of "either way add the resourceId, version and type and any other relevant data"